src/Microsoft.PackageManagement.MsiProvider/project.json

```diff
@@ -3,7 +3,8 @@
 	"version": "1.0.0-*",
 
 	"buildOptions": {
-		"warningsAsErrors": true
+		"warningsAsErrors": true,
+                "debugType": "portable"
```

Comment:
The project `MsiProvider` seems doesn't get built for .NET Core since there is no `netstandard1.6` target framework section here. For projects like this, the `debugType` setting doesn't apply.